### DISCUSSION TITLE
Where can I find ` _vercel.testing.app`?

### DISCUSSION THREAD

**Discussion by OneFabric** [vercel/community#6276] [Help] [answered]
### Summary

I am supposed to edit ` _vercel.testing.app` with a TXT record to use a certain domain. However, I am not sure what `_vercel.testing.app` is, and where I can find it to edit it. This domain is a new domain I created in my project. The older one works, and is _not_ giving me an error. Details below.

### Steps to Reproduce

I am pretty certain this issue is happening since I am new to vercel. I am not familiar with the infrastructure, and am having trouble finding documentation for the definition of `_vercel.testing.app`.

So, instead of giving repro steps, I thought I would explain the use case. I added a new domain for testing in my project. This domain gives me an error called 'Domain is Pending Verification'

Any documentation or explanation on how I can find `_vercel.testing.app` so I can enable the TXT record would go a long way.

I found [this Github discussion](https://github.com/orgs/vercel/discussions/1460) that deals with the issue, but I am not sure if my particular problem is with Vercel or DNS technology in general.

Thank you for your time! I may not respond soon, but will get back to you as soon as I can.

**Comment by amyegan** [comment, marked as answer]
Sometimes a domain is already associated with a different account for one reason or another. In that case, you can [verify domain access](https://vercel.com/docs/projects/domains/add-a-domain#verify-domain-access) using a TXT record. 

If you're trying to add the domain `testing.app` and you've already set the appropriate A and CNAME records, then you would need to go back to your domain registrar and add the TXT record for the `_vercel.testing.app` subdomain the same way. 

https://vercel.com/docs/projects/domains/add-a-domain#verify-domain-access

**Reply by OneFabric** [reply]
I didn't realize that my problem is specific to my understanding of internet infrastructure. Looks like Vercel is just highlighting the problem, but is not responsible for it.

Is this understanding correct? If yes, I would like to close as answered / unrelated to vercel specifically. 

Thanks for your time!
